packages/terracotta-react/src/utils/getData.ts

```diff
@@ -0,0 +1,22 @@
+export interface GetDataPayload {
+	host: string
+	endpoint: string
+	params?: Record<string, string | boolean | number>
+	additionalHeaders?: Record<string, string>
+}
+
+export default async ({
+	host,
+	endpoint,
+	params = {},
+	additionalHeaders = {},
+}: GetDataPayload): Promise<any> => {
```

Comment:
Guessing the response can't really be typed?

Reply:
I would like to type it but I haven't found a way yet. Tried using <T>, but maybe I am not that familiar with the syntax. I would like to be able to use the function and somehow pass the ResponseType that I expect every time I use getData.

Reply:
Thank you for the valuable suggestions 🚀

Reply:
https://github.com/DHI/react-components/blob/master/packages/react-components-lab/src/hooks/useLocalStorage.ts Maybe this can be used as an example for it?

Reply:
I found it, pushed the commit

Reply:
I wanted the arrow function hehe, here https://stackoverflow.com/questions/32308370/what-is-the-syntax-for-typescript-arrow-functions-with-generics